operator/src/main/java/oracle/kubernetes/operator/Main.java

```diff
@@ -578,6 +595,17 @@ public NextAction apply(Packet packet) {
     }
   }
 
+  /**
+   * Adds the specified namespace to the list of started namespaces, returning true if it was not
+   * already present.
+   * @param ns the namespace name
+   * @return false if the namespace had already been started
+   */
+  static boolean addStartedNamespace(String ns) {
+    AtomicBoolean a = isNamespaceStarted.computeIfAbsent(ns, (key) -> new AtomicBoolean(false));
+    return !a.getAndSet(true);
```

Comment:
this line seems overly complicated and prone to future update errors

Reply:
This is just a refactoring of the old code in line #567.  The pattern is used in other places.